krux_boto/__init__.py

```diff
@@ -77,7 +77,7 @@ def __init__(self, logger = None, stats = None, parser = None):
         self.___parser = parser or get_parser(description = self.___name)
 
         ### in case we got some of the information via the CLI
-        self.___args    = self.___parser.parse_args()
+        self.___args    = self.___parser.parse_args(args = cli_args)
```

Comment:
Hi @anishbhuta, I know we should not leave space between variable and = in argument assignment. And I did this for 2 purposes: 1) follow the code styling used in this class, 2) make less  change as I can, then @jib can have a clear view about what did I modify. 

Then, I will fix the code styling issue for both my code and the existing code in the next commit.